modules/compat-libraries-32bit.yml

```
@@ -0,0 +1,26 @@
+x-compat-i386-opts: &compat_i386_opts
+  prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+  ldflags: -L/app/lib32
+  prepend-path: /usr/lib/sdk/toolchain-i386/bin
+  env:
+    CC: i686-unknown-linux-gnu-gcc
+    CXX: i686-unknown-linux-gnu-g++
+    libdir: /app/lib32
```

`libdir` here is supposedly not an env var, but a build options, which tells the build system where to install libraries, similarly to `prefix`.
```suggestion
  libdir: /app/lib32
```